cmd/tk/workflow.go

```diff
@@ -193,7 +191,10 @@ Otherwise run tk show --dangerous-allow-redirect to bypass this check.`)
 			return err
 		}
 
-		pageln(pretty.String())
+		if err := pageln(pretty.String()); err != nil {
+			return err
+		}
+
 		return nil
```

Comment:
If it's the last statement, no need for if-else, as returning err does the same thing